docs/production/upgrade-or-modify.md

```diff
@@ -133,6 +133,14 @@ suggest using that updated template to update
    section in `/etc/zulip/settings-new.py` and copy the setting from
    `settings.py` into there.
 
+   The following tool may help, by finding the most likely version of
+   the template that your `/etc/zulip/settings.py` was installed
+   using, and the differences that your file has from that:
+
+   ```
+   /home/zulip/deployments/current/scripts/minimal-template-diff
```

Comment:
Let's move this to `scripts/setup/` -- we try to keep `scripts/` for things that have good `--help` and are things we expect users to run routinely.

It may also want a clearer name; maybe `compare-settings-to-template` or something?